paf-mvp-operator-express/src/operator-node.ts

```diff
@@ -765,14 +765,26 @@ export class OperatorNode extends Node {
 
   catchErrorsWithRedirectToReferer =
```

Comment:
I plan to remove this method in the next PR: this must be centralized in `catchErrors` for consistency and to avoid duplication.